src/include/denc.h

```diff
@@ -307,6 +349,14 @@ inline void denc_signed_varint(T& v, bufferptr::iterator& p)
 inline void denc_varint_lowz(uint64_t v, size_t& p) {
   p += sizeof(v) + 2;
 }
+
+inline void denc_varint_lowz_exact(uint64_t v, size_t& p) {
+  int lowznib = v ? (ctz(v) / 4) : 0;
+  if (lowznib > 3)
+    lowznib = 3;
+  v >>= lowznib * 4 - 2;
```

Comment:
FIXME(micro optimisation): AFAIK shifts are slightly more expensive than additions/plain math on x86. Maybe we could apply here similar idiom like in the case of `denc_lba_exact`.